Prototype HUD

TBAce11 opened the pull request:
Ce pull request contient la base du HUD qui sera modifié au fûr et à mesure que le projet progresse. Ainsi, vous pourrez me faire part des détails à modifier une fois qu'il sera inclus dans le master. De plus, comme exigé par Cristian, il est important qu'il soit approuvé d'ici dimanche afin d'éviter des contradictions que pourrait engendrer le grimoire.

P.S.: Soit attentif aux fichiers autres que Gamecore et PlayState, Cristian. Tu as effectué des modifications rapides en classe qui risquent de poser problème avec tes menus.

TBAce11 committed:
Update 26/02/2017

-Ajout de tout les composantes essentielles au HUD sur la liste d'idées
(bouton grimoire?)
-Remplissage des deux barres à l'aide de rects transparents
-Niveau de transparence et placement des éléments ainsi que l'apparence
du HUD des barres seulement à discuter

TBAce11 committed:
Update 26/02/2017

-Modification de l'emplacement et de la taille des composantes HUD
-Debug en bas à gauche

TBAce11 committed:
Update 01/03/2017

-Finalisation de l'apparence du HUD + détails à ajouter éventuellement
-Ajout des classes SpellBookState et Potions
-Corrections mineures un peu partout dans le projet effectuées en classe
-Ajout des fichiers PNG et Photoshop dans les ressources

Note:
-Positionnement en x du drawString à corriger

TBAce11 committed:
Update 12/03/2017

-Changements établis selon les conseils de Cristian

TheGreatMarkus commented:
Um tarik faut que tu met à jour ta branche puis que tu implemente les changements demandé avant quon puisse parler d'un pull request
Pour le moment ta branche n'es même pas sable te il y a des conflits à résoudre.

cristian-aldea commented on Spellington/src/ca/qc/bdeb/info204/spellington/gameentities/Spellington.java:
```diff
@@ -36,9 +36,8 @@
-    public static float sLifePoint;
-    private static final int MASS = 2;
-    private static final Vector2D X_ACC = new Vector2D(0.003f, 0);
+    private static final int MASS = 1;
```
Enlève cette variable MASS. Elle n'avait pas été enlevé quand t'as fais ta mise à jour avec le master

Commit:
Update 15/03/2017

Changement
-Changement de la gestion du fontPaladin
-Changements legers de la disposation du HUD

cristian-aldea reviewed (approved):
Tout est bien pour le merge de cette branche avec le master.
